### ISSUE TITLE
[BUG] Admin commands not checking perms properly

### ISSUE THREAD

**Issue by KOOKIIEStudios** [KOOKIIEStudios/PalCON-Discord#18]
**Describe the bug**
![initial report](https://github.com/KOOKIIEStudios/PalCON-Discord/assets/25145447/c544ce14-8048-40ab-a226-bd59181b650c)

**To Reproduce**
1. Have a non-admin use any commands other than `/info` or `/online`
2. Notice that they're somehow able to use them
  ![testing on our side](https://github.com/KOOKIIEStudios/PalCON-Discord/assets/25145447/5aeacefd-ba3d-4076-920a-0f786099f956)

**Expected behavior**
Non-admins should not be allowed to use admin commands

**Plausible explanations**
The way this is currently handled is the following in line 5 of `main.py`:
```py
from discord.ext.commands import has_permissions
```
which is then consumed as a decorator for admin commands like so:
```py
@has_permissions(administrator=True)
```

Based on the [latest API docs for Interactions (which slash commands fall under) from discord.py](https://discordpy.readthedocs.io/en/latest/interactions/api.html?highlight=default_permission#discord.app_commands.checks.has_permissions), the way to check for permissions is to use this instead:
```py
@app_commands.checks.has_permissions(administrator=True)
```
Note that we already import `app_commands` in line 4 of `main.py`:
```py
from discord import app_commands
```

This solution has been tested by the reporter of the bug. We'll just need to perform the refactoring to remote for this issue.
![findings following investigation](https://github.com/KOOKIIEStudios/PalCON-Discord/assets/25145447/aac6a108-8044-41fc-90ef-2ec8e8411b6a)

**Comment by alexraskin** [comment]
I would like to take this.

**Comment by Bratah123** [comment]
> I would like to take this.

Assigned! Feel free to send a PR, i'd gladly review and approve

**Comment by Bratah123** [comment]
Issue fixed in #20

Thanks @alexraskin!
